docs/README.md

```diff
@@ -2,8 +2,82 @@
 
 # Kyodo
 
-**Kyodo**
-Ethereum blockchain based web app, the cross-platform incentive tool for building the crypto economy for DAO based on Impression revenue model. We build Kyodo to run Decent.Fund.
+## Installation
+
+This is a monorepo for Kyodo project consisting of 3 modules:
+
+- Kyodo web app to setup colony, sign final tips, get your balance and view tasks progress
+- Kyodo backend to run mongodb server and tipping watch within a period
+- Kyodo bot to run riot bot for task creation and instant tipping
+
+## Get started
+
+### Prerequisites
+
+- Yarn
+- Docker
+- Node 9.10
+
+### To setup monorepo packages and its dependencies
+
+```
+git clone https://github.com/decentfund/kyodo.git
+cd kyodo
+yarn
+```
+
+### Add initial distribution (optional)
+
+```
+# Create initial distribution of tokens
+# Change `deploy_parameters.json` to reflect the initial distribution you want or leave the accounts section empty.
+cp migrations/deploy_parameters.example.json migrations/deploy_parameters.json
+```
+
+
+### Setup contracts
+
+```bash
+cd packages/kyodo-contracts
```

Comment:
What do you think about this starting [approach](https://github.com/JoinColony/colonyStarter/blob/master/package.json)? 
It eliminates the need to navigate to the `packages/kyodo-contracts` folder and allows the developer to launch all the scripts from the project root